### DISCUSSION TITLE
Import/export settings

### DISCUSSION THREAD

**Discussion by ygoe** [pawelsalawa/sqlitestudio#4885] [Q&A]
### Details
I am installing SQLite Studio on another computer and don't want to manually configure everything from scratch. Instead, I want to copy the application configuration from one computer to another. (And I would like to include it in my file backup as well. I'm not making full system image backups.)

I've looked for a menu item or other command in the GUI but couldn't find any. I've also looked for the settings in a file or the Windows registry but also couldn't find any. Where is that data stored and how can I copy it to another computer? Maybe this should be added as a feature or at least documented in the Wiki. Otherwise, I might ask here again next time I do this.

**Comment by pawelsalawa** [comment]
https://github.com/pawelsalawa/sqlitestudio/wiki/Configuration_directory_location

**Comment by ygoe** [comment]
Thank you for the reply. But the information provided there is incorrect. It's not %appdata% but %localappdata%. There I found the file and could copy over the settings.
